kubernetes-py-exposed-deployment/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pulumi==3.11.0
```

Comment:
we usually use ranges than go between major versions so that we don't need to keep bumping this :)